hirola/front/tests/test_users.py

```diff
@@ -206,3 +206,46 @@ def test_remember_user_html_rendered(self):
         '''
         response = self.client.get("/login")
         self.assertContains(response, "name=\"remember-user\"")
+
+
+class UserChangeRegistrationEmail(BaseTestCase):
+    def setUp(self):
+        super(UserChangeRegistrationEmail, self).setUp()
+
+    def test_successful_change_of_registration_email(self):
+        '''
+        Test that when a user serves correct input data to the signup form:
+            - The user is added to the database.
+            - The user is visible in the admin page.
```

Comment:
The comments may not depict what is on the test:
- To test user visibility on the admin page you have to give the test client admin rights and forcefully log him or her in.
- A test for landing page redirection would be asserting a 302 status code and a URL for the landing page.